awesomplete.js

```diff
@@ -42,15 +43,18 @@ var _ = function (input, o) {
 
 	this.ul = $.create("ul", {
 		hidden: "hidden",
+        role: "listbox",
+        id: "awescomplete_list",
```

Comment:
This won't work very well if you have multiple instances of Awesomplete (unless I'm missing something...)